apps/browser/qml/pages/PermissionPage.qml

```diff
@@ -60,24 +60,37 @@ Page {
                                         title: qsTrId("sailfish_browser-ti-geolocation"),
                                         type: "geolocation",
                                         capability: PermissionManager.Prompt,
-                                        iconSource: "image://theme/icon-m-browser-location"
+                                        iconSource: "image://theme/icon-m-browser-location",
+                                        isSensitiveData: true
```

Comment:
```suggestion
                                        sensitiveData: true
```
It is preferable to not to use the suffix in qml for boolean

Reply:
Edited. Thanks.